proxies/nodejs/handlers/client.ts

```diff
@@ -15,9 +15,16 @@ export const handleClient = async (ctx: Koa.ParameterizedContext, clients: { [ke
       error: "Invalid request: missing clientId"
     }
   } else {
-    const client = initialize(body.sdkKey, body.options)
-    clients[body.clientId] = client
-    ctx.status = 201
-    ctx.set('Location',`client/${body.clientId}`)
+    try {
+      const client = initialize(body.sdkKey, body.options)
+      clients[body.clientId] = client
```

Comment:
I was trying to recreate a test case when the catch will be called but it did not so i am not sure if this helps

Reply:
We throw an error if there is no sdk key. I believe we might want to test that case so that's why the try-catch is here. If something actually goes wrong during initialization we log so you're right this won't catch that.